modules/network/traffic-manager/README.md

```diff
@@ -17,9 +17,12 @@ The module also allows for the configuration of monitor settings such as protoco
 
 | Name                    | Type     | Required | Description                                                                                                                                                       |
 | :---------------------- | :------: | :------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `prefix`                | `string` | No       | Prefix of Traffic Manager Profile Name                                                                                                                            |
-| `name`                  | `string` | No       | Name of Traffic Manager Profile Resource                                                                                                                          |
-| `trafficManagerDnsName` | `string` | No       | Relative DNS name for the traffic manager profile, must be globally unique.                                                                                       |
+| `name`                  | `string` | Yes      | Name of Traffic Manager Profile Resource                                                                                                                          |
+| `trafficManagerDnsName` | `string` | Yes      | Relative DNS name for the traffic manager profile, must be globally unique.                                                                                       |
```

Comment:
Making optional parameters required is a breaking change (see [documentation](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#bumping-major-version)). Can you bump the major version in `version.json` and update the module version in examples?